WikipediaUnitTests/Code/WMFBlacklistTests.m

```diff
@@ -0,0 +1,53 @@
+//
+//  WMFBlacklistTests.m
+//  Wikipedia
+//
+//  Created by Corey Floyd on 1/21/16.
+//  Copyright © 2016 Wikimedia Foundation. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "WMFAsyncTestCase.h"
+#import "WMFRelatedSectionBlackList.h"
+#import "MWKTitle.h"
+
+@interface WMFRelatedSectionBlackList (WMFTesting)
+
++ (instancetype)loadFromDisk;
+
+@end
+
+
+@interface WMFBlacklistTests : WMFAsyncTestCase
+
+@end
+
+@implementation WMFBlacklistTests
+
+- (void)tearDown {
+    WMFRelatedSectionBlackList* bl = [[WMFRelatedSectionBlackList alloc] init];
+    [bl removeAllEntries];
+    [bl save];
+    [super tearDown];
+}
+
+- (void)testPersistsToDisk {
+    PushExpectation();
+    MWKTitle* title                = [[MWKTitle alloc] initWithString:@"some-title" site:[MWKSite siteWithCurrentLocale]];
+    WMFRelatedSectionBlackList* bl = [[WMFRelatedSectionBlackList alloc] init];
+    [bl addEntry:title];
+    [bl save].then(^(){
```

Comment:
`XCTestCase+PromiseKit.h` is provided to simplify async testing w/ Promises—no need to push/pop/wait for expectations.